wdl/Tasks04.wdl

```diff
@@ -394,20 +415,30 @@ task CountPE {
 
     set -euo pipefail
     svtk vcf2bed --split-bnd --no-header ~{vcf} test.bed
-    awk -v OFS="\t" -v window=5000 '{if ($2-window>0){print $1,$2-window,$2+window}else{print $1,0,$2+window}}' test.bed  >> region.bed
-    awk -v OFS="\t" -v window=5000 '{if ($3-window>0){print $1,$3-window,$3+window}else{print $1,0,$3+window}}' test.bed  >> region.bed
+    awk -v OFS="\t" -v window=500 '{if ($2-window>0){print $1,$2-window,$2+window}else{print $1,0,$2+window}}' test.bed  > region.bed
+    awk -v OFS="\t" -v window=500 '{if ($3-window>0){print $1,$3-window,$3+window}else{print $1,0,$3+window}}' test.bed  >> region.bed
```

Comment:
Is 500 bases of padding always sufficient to get the evidence we needed? 

Should we parameterize this value? I could also see eventually using a value computed from the data (based on insert size metrics distributions, for example).

Reply:
500 is default the "outer" padding for `svtk count-pe` so hard-coding is okay in this case. 

It would be better to adjust this per-batch based on insert size distribution, but this would take a bit of work to optimize. It's not obvious how to do that given that we don't usually batch based on the mean or variance of the insert size distribution, although they may correlate with dosage score in some way.